bgpd/bgp_routemap.c

```diff
@@ -5708,6 +5762,35 @@ DEFPY_YANG(
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY_YANG(
+	match_community_limit, match_community_limit_cmd,
+	"[no$no] match community-limit [(1-1024)$limit]",
+	NO_STR MATCH_STR "Match BGP community limit\n"
```

Comment:
Please use the format as any other (every new token with a new line). E.g.:

```
NO_STR
MATCH_STR
...
```

Reply:
done